package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@koopjs/koop-provider-hub-search",
-  "version": "2.1.1",
+  "version": "3.0.0",
```

Comment:
don't change the version number in this PR. We merge to master, then we will do a specific release commit on master where we update the package.json.

Reply:
Got it! I have reverted the version bump. Will update version once merged.